src/drivers/net/emac/emac.c

```diff
@@ -723,7 +650,8 @@ static int ti816x_init(void) {
 	}
 
 	nic->drv_ops = &ti816x_ops;
-	nic->irq = nic->base_addr = 0;
+	nic->irq = MACRXTHR0;
```

Comment:
Btw, as I see `irq` field is not used anywhere at all. I don't mean that this line should be removed, just for the note.